apis/pkg/v1alpha1/config.go

```diff
@@ -25,6 +25,12 @@ import (
 // Values provided will override package manager defaults. Labels and
 // annotations are passed to both the controller Deployment and ServiceAccount.
 type ControllerConfigSpec struct {
+	// Number of desired pods. This is a pointer to distinguish between explicit
+	// zero and not specified. Defaults to 1.
+	// Note: If number more than 1 is set, environment variable "LEADER_ELECTION"
+	// must exist and must be true.
```

Comment:
Could you adjust this note to just mention that if more than 1 replica is set for a controller and leader election is not enabled then controllers could conflict?